### PR TITLE
add: add documentation on how to start devcontainer and application

### PR DESCRIPTION
**Pull request by admantiumblack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
...
+
+3. Press "reopen in container" on vscode.
+
+    > **Note**: port 8501 should be available.
+
+## How to start application
+
+1. enter the command
+
+    ``
```
Comment (elvanselvano71) [inline README.md]: Add shell annotation

File: README.md [hunk 2]
```diff
@@ -1 +1,23 @@
-# streamlit-whisper
\ No newline at end of file
+# streamlit-whisper
+
+## How to use:
+
+1. Clone repository
```
Comment (elvanselvano71) [inline README.md:5]: Just to clarify, do we need to install requirements.txt? Also, user need Python installed right?

Comment (elvanselvano71) [reply README.md:5]: If yes, please add in new section called tools

Comment (admantiumblack) [reply README.md:5]: No, all handled by devcontainer

**Pull request closed without merging**